src/test/compile-fail/issue-53300.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2012-2014 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Can you turn this into an `ui` test instead? We're slowly moving away from `compile-fail`.

Reply:
K, will do. Thanks. :+1: